### ISSUE TITLE
Script fails

### ISSUE THREAD

**Issue by geotheory** [tdh8316/triangler#11]
Broken for me:

```
>>> triangler_instance.convert_and_save("input.jpg", "output.jpg")
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/opt/anaconda3/lib/python3.8/site-packages/triangler/mod.py", line 89, in convert_and_save
    im = Image.fromarray(self.convert(source).astype(np.uint8))
  File "/opt/anaconda3/lib/python3.8/site-packages/triangler/mod.py", line 58, in convert
    return process(
RuntimeWarning: Option -l or --reduce is temporarily unavailable due to the deprecated parameter multichannel of scikit-image.
```

**Comment by tdh8316** [comment]
Hi, thanks for letting me know about the issue.
I temporarily removed a `pyramid_reduce` option because of the dependency on the deprecated method.

You can disable this option by setting the `pyramid_reduce` parameter to `False`. For example:
```
# Create Triangler instance
triangler_instance = triangler.Triangler(
    pyramid_reduce=False,
)
```

This should be fixed soon. I'm a little busy so I'm not sure I can tackle this issue for now, however.
But I'll fix it. Thanks for reporting the issue!

Best regards,

**Comment by tdh8316** [comment]
Fixed at commit 8f77b8181693913e5b5a94538498ea6102761550.
